controllers/admin/AdminTranslationsController.php

```diff
@@ -83,7 +86,8 @@ public function __construct()
 
         parent::__construct();
 
-        $this->link_lang_pack = str_replace('%ps_version%', _PS_VERSION_, $this->link_lang_pack);
+        $language_pack_loader = $this->container->get('prestashop.core.language.pack.loader.remote');
```

Comment:
Just a warning here, you get the service from the container but it's not defined in the legacy container I guess since it's a controller from BO it works fine because the Symfony container is always used in the BO anyway But this wouldn't work on a front controller just FYI